React/Views/SafeAreaView/RCTSafeAreaShadowView.m

```diff
@@ -21,12 +22,23 @@ - (void)setLocalData:(RCTSafeAreaViewLocalData *)localData
 
   UIEdgeInsets insets = localData.insets;
 
-  super.paddingLeft = (YGValue){insets.left, YGUnitPoint};
-  super.paddingRight = (YGValue){insets.right, YGUnitPoint};
   super.paddingTop = (YGValue){insets.top, YGUnitPoint};
   super.paddingBottom = (YGValue){insets.bottom, YGUnitPoint};
 
-  [self didSetProps:@[@"paddingLeft", @"paddingRight", @"paddingTop", @"paddingBottom"]];
+  if ([[RCTI18nUtil sharedInstance] doLeftAndRightSwapInRTL]) {
+    if ([[RCTI18nUtil sharedInstance] isRTL]) {
```

Comment:
... and btw this is not proper way to check for RTL. That prop return a global setting which is not so useful because we have `direction` style.

And, actually, there is not way to check it properly here because the computed value of `display` prop depends on outer nodes and unabailable here.

So, see my next comment.